benchmarks/release_benchmarking/release/v3.0.0/v3.0.0.md

```diff
@@ -0,0 +1,222 @@
+
+For each configuration, we ran several instances of the application pipeline in parallel to evaluate performance under load. Up to 8 instances ran in parallel on the x86 platform under tests, while IGX machines were limited to 3 concurrent instances.
+
+### Steps to Reproduce
+
+1. Use Git to check out HoloHub commit 6e4e377a to match the benchmarked versions of the Endoscopy Tool Tracking and Multi-AI Ultrasound applications.
+2. Follow the [Release Benchmarking project instructions](/benchmarks/release_benchmarking/README.md) to reproduce the benchmarking procedure.
+
+## Results
+
+End-to-end latency results are reported in milliseconds (ms). Vertical axes are scaled relative to results for each platform.
+
+### Executive Summary
+
+Overall, there is considerable improvement in performance compared to v2.6 in the experiments with "As Fast As Possible" mode, especially in the case of the x86 setup. We additionally tested on the x86-Ada6000 configuration with CUDA 12.8 and the 570 driver, which, though not officially supported in this release, showed good performance improvements (up to 2x performance improvement over x86-A6000). Similar to the previous reports, we see better performance improvements for a smaller (1-3) number of instances, though we do not observe significant improvements in the real-time cases.
+
+#### Comparison: Endoscopy Tool Tracking in "As Fast As Possible" Mode
+
+
+#### Comparison: Endoscopy Tool Tracking in "As Fast As Possible" Mode
+| Statistic                   | x86_64 (A6000)                      | IGX iGPU                              | IGX dGPU (A6000)                    | IGX dGPU (ADA6000) |
+|--------------------------   |-----------------                    |----------                             |----------                           |------------------|
+| **Average E2E Latency**     |   Best case: no change              |  Best case: decreased by upto 9%      |  Best case: increased by 1.4%       | Best case: decreased by upto 2.7% |
+|                             | Range: +2.9% to 0%                  | Range: -0.8% to -9%                   |     Range: +4.3% to +1.4%           |  Range: +2.3% to -2.71%                |
+| **99.9 Percentile Latency** |  Best case: decreased by up to 29%  |  Best case: decreased by upto 0.8%    |  Best case: decreased by upto 26%   | Best case: decreased by upto 31% |
+|                             | Range: -11% to -29%                 | Range:  -0.2% to -0.8%                |  Range: +3.8% to -26%               | Range: -25% to -31%              |
+| **Max Percentile Latency**  |  Best case: decreased by up to: 72% |  Best case: decreased by upto 5%      |  Best case: increased by 16%        | Best case: decreased by upto 56%|
+|                             |  Range: +33% to -72%                | Range:   +46% to -5%                  |   Range: +58% to +16%               |  Range: +55% to -56%               |
+### Endoscopy Tool Tracking with Real-Time Video Input
```

Comment:
For a future benchmarking investigation: real-time latencies are reported with a floor of ~33ms matching the replayer setting at 30fps, both here and in past benchmarking reports. I wonder if we could improve the flow benchmarking tools (and/or the video replayer operator) in the future to report the "real" latency of message propagation through each pipeline, excluding the periodic wait condition. No change necessary for v3.0 results.